bin/phpunit/test.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const { spawnSync } = require( 'child_process' );
+
+/**
+ * Internal dependencies
+ */
+const { plugins } = require( '../../plugins.json' );
+const { formats } = require( '../plugin/lib/logger' );
+
+const args = process.argv.slice( 2 );
+const { WPP_PLUGIN, WPP_MULTISITE } = process.env;
+const pluginBasePath = path.resolve( __dirname, '../../' );
+const pluginBaseName = path.basename( pluginBasePath );
+const pluginsDir = path.resolve( pluginBasePath, 'plugins' );
+const phpunitBin = path.resolve( pluginBasePath, 'vendor/bin/phpunit' );
+const wpEnvBin = path.resolve( pluginBasePath, 'node_modules/.bin/wp-env' );
+const wpPluginsDir = `/var/www/html/wp-content/plugins/${ pluginBaseName }`;
+
+const _plugins = []; // Store absolute paths to plugins.
+
+if ( WPP_PLUGIN ) {
+	if (
+		'performance-lab' !== WPP_PLUGIN &&
+		! plugins.includes( WPP_PLUGIN )
+	) {
+		// eslint-disable-next-line no-console
+		console.error(
+			formats.error(
+				`The plugin ${ WPP_PLUGIN } is not a valid plugin managed as part of this project.`
+			)
+		);
```

Comment:
Can we utilize `log` from `require( '../plugin/lib/logger' )`?

```suggestion
		log(
			formats.error(
				`The plugin ${ WPP_PLUGIN } is not a valid plugin managed as part of this project.`
			)
		);
```

Reply:
I think better to use `console.error()` as it goes to `STDERR` instead of `STDOUT`.

Otherwise, this could be added to logger as an alias in the same way that was done for `console.log`.